packages/ui-tag/src/Tag/index.tsx

```diff
@@ -57,27 +59,25 @@ class Tag extends Component<TagProps> {
     readOnly: false
   }
 
+  ref: Element | null = null
+
   componentDidMount() {
     this.props.makeStyles?.()
   }
 
-  // @ts-expect-error ts-migrate(6133) FIXME: 'prevProps' is declared but its value is never rea... Remove this comment to see the full error message
-  componentDidUpdate(prevProps, prevState, snapshot) {
+  componentDidUpdate() {
     this.props.makeStyles?.()
   }
 
   get focused() {
-    // @ts-expect-error ts-migrate(2339) FIXME: Property '_container' does not exist on type 'Tag'... Remove this comment to see the full error message
-    return isActiveElement(this._container)
+    return isActiveElement(this.ref)
   }
 
   focus = () => {
-    // @ts-expect-error ts-migrate(2339) FIXME: Property '_container' does not exist on type 'Tag'... Remove this comment to see the full error message
-    this._container && this._container.focus()
```

Comment:
This seemed to be a bug since the container would be View, and it doesn't have a focus method. So I ended up using the ref element and try to focus that. Also it doesn't seem that it is used anywhere and a normal (span) Tag cannot be focused anyway...

Reply:
Well it can be used from the outside, and since the button version of the Tag CAN be focused, there might be someone using this.